src/index.ts

```diff
@@ -19,4 +19,4 @@ const cardValidator = {
   postalCode,
 };
 
-export = cardValidator;
+export default cardValidator;
```

Comment:
Does changing this to have a default export change the way consumers of this module import it?

Reply:
Yes, you can see the change in the `__tests__/credit-card-type.ts` file

Before: `import cardValidator = require("../")`
After:`import cardValidator from "../"`

Reply:
Got it! Would this be a breaking change?

Reply:
same concern here, would this be a breaking change?